gradle.properties

```diff
@@ -10,9 +10,9 @@ releasesRepoUrl=https://repo.datastax.com/artifactory/datastax-public-releases-l
 avroVersion=1.10.2
 lombokVersion=1.18.20
 ossDriverVersion=4.15.0
-cassandra3Version=3.11.10
```

Comment:
Could you please revet those as well?

Reply:
@aymkhalil are these causing any build failures?

Reply:
No, but event the CI ones where not causing breaking issues. The idea was to keep this PR scope limited to docs so we can discuss dependency version independently and unblock this PR faster.